lib/manager/containers/ActiveFeedVersionNavigator.js

```diff
@@ -72,15 +73,22 @@ const mapStateToProps = (state: AppState, ownProps: Props) => {
     feedVersions.length >= feedVersionIndex
   ) {
     version = sortedVersions[feedVersionIndex - 1]
+
+    if (gtfs.filter.comparedVersion) {
+      comparedVersionIndex = sortedVersions.findIndex(feedVer => feedVer.id === gtfs.filter.comparedVersion) + 1
+      comparedVersion = sortedVersions[comparedVersionIndex - 1]
+    }
   }
   return {
     gtfs,
     hasVersions,
     feedVersionIndex,
+    comparedVersionIndex,
     gtfsPlusValidation: state.gtfsplus.validation,
     sortedVersions,
     user: state.user,
     version,
+    comparedVersion,
     versionIndexDoesNotExist,
     versionSection: subpage
   }
```

Comment:
These props are starting to build up quite a bit. It might be a good time to split up this container so that each individual component becomes a connected component to avoid passing props around so much.

Reply:
I'll let @landonreed decide whether to make that a blocker for this PR.

Reply:
Seems a bit out of scope for this PR, but probably worth dropping a `FIXME: prop drilling` in the files that need a refactor.